DigitalOffice/Tests/UserServiceTests/ValidatorsTests/UserCredentialsValidatorTests.cs

```diff
@@ -0,0 +1,62 @@
+﻿using FluentValidation;
+using FluentValidation.TestHelper;
+using NUnit.Framework;
+using UserService.Database.Entities;
+using UserService.Models;
+using UserService.Validators;
+
+namespace UserServiceUnitTests.ValidatorsTests
+{
+    class UserCredentialsValidatorTests
+    {
+        private IValidator<UserCredentials> validator;
+
+        [SetUp]
+        public void Initialization()
+        {
+            validator = new UserLoginValidator();
+        }
+
+        [Test]
+        public void EmptyLoginEmptyPasswordTest()
+        {
+            validator.ShouldHaveValidationErrorFor(x => x.Email, "");
+            validator.ShouldHaveValidationErrorFor(x => x.Password, "");
+        }
+
+        [Test]
+        public void EmptyLoginTest()
+        {
+            validator.ShouldHaveValidationErrorFor(x => x.Email, "");
+            validator.ShouldNotHaveValidationErrorFor(x => x.Password, "Example");
+        }
+
+        [Test]
+        public void EmptyPasswordTest()
+        {
+            validator.ShouldNotHaveValidationErrorFor(x => x.Email, "Example@mail.com");
+            validator.ShouldHaveValidationErrorFor(x => x.Password, "");
+        }
+
+        [Test]
+        public void EmailTooLongTest()
+        {
+            validator.ShouldHaveValidationErrorFor(x => x.Email,
+                "Exampleeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee@mail." +
+                "coooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooom");
```

Comment:
use stringbuilder for this

Reply:
fixed

Reply:
no, you should use `new string(...` for this

Reply:
fixed